XRTK-Core/Packages/com.xrtk.core/Utilities/Editor/PathFinder.cs

```diff
@@ -1,4 +1,7 @@
-﻿using UnityEngine;
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
This is ours not Microsofts.

Reply:
Ahh right, well I was just not removing it, curious as I remember you adding it.

Reply:
I read that wrong, Fixed